src/smokeTest/java/uk/gov/hmcts/reform/sendletter/controllers/CreateLetterSmokeTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.sendletter.controllers;
+
+import io.restassured.RestAssured;
+import io.restassured.specification.RequestSpecification;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+
+import java.util.UUID;
+
+public class CreateLetterSmokeTest extends SmokeTestSuite {
+
+    @Test
+    public void should_create_letter_successfully() throws Exception {
+
+        String jwt = signIn();
+
+        String letterJson = new JSONObject()
+            .put("type", "smoke_test")
+            .put("documents", new JSONArray()
+                .put(new JSONObject()
+                    .put("template", "some_template")
+                    .put("values", new JSONObject()
+                        .put("a", "b")
+                        .put("uuid", UUID.randomUUID().toString()) // so that we don't send the same letter twice
+                    )
+                )
+            ).toString();
+
+        String id = givenJwt(jwt)
+            .and()
+            .body(letterJson)
+            .when()
+            .post("/letters")
+            .then()
+            .statusCode(200)
+            .extract()
+            .body()
+            .jsonPath()
+            .get("letter_id");
+
+        givenJwt(jwt)
+            .when()
+            .get("/letters/" + id)
+            .then()
+            .statusCode(200);
+    }
+
+    private RequestSpecification givenJwt(String jwt) {
+        return RestAssured
+            .given()
```

Comment:
will it work without `relaxedHTTPSValidation()`?

Reply:
added